audio_chain_example.py

```diff
@@ -0,0 +1,38 @@
+import langchain
```

Comment:
can we add this as a notebook in `docs/modules/chains/examples`? that way will auto get put in documentation

Reply:
I'll do this tomorrow.

Reply:
sg